qgispluginci/parameters.py

```diff
@@ -22,15 +22,18 @@ class Parameters:
     project_slug: str
         The project slug on SCM host (e.g. Github) and translation platform (e.g. Transifex).
         Not required when running on Travis since deduced from `$TRAVIS_REPO_SLUG`environment variable.
-        Otherwise, defaults to
 
     transifex_coordinator: str
         The username of the coordinator in Transifex.
         Required to create new languages.
 
     transifex_organization: str
         The organization name in Transifex
-        Defaults to: `organization`
+        Defaults to: the GitHub organization slug
+
+    transifex_project_slug: str
```

Comment:
if I'm correct this should be renamed to just "transifex_project" (slug is organisation + project)

Reply:
It seems there is a mismatch somewhere... :)

If I'm following examples
https://github.com/opengisch/qgis-plugin-ci/blob/master/.qgis-plugin-ci#L4..L5
```
github_organization_slug: opengisch
project_slug: qgis-plugin-ci
```
Is-it not the URL to reach https://github.com/opengisch/qgis-plugin-ci ?

But it's true, in the docs, you are mentioning that 

>  github_organization_slug: str
        The organization slug on SCM host (e.g. Github) and translation platform (e.g. Transifex).
        Not required when running on Travis since deduced from `$TRAVIS_REPO_SLUG`environment variable.

On travis:
> TRAVIS_REPO_SLUG: The slug (in form: owner_name/repo_name) of the repository currently being built.

So if I'm correct, there was a mismatch before, no?

Looking at this one too for instance. The slug is never organization + project name.
https://github.com/opengisch/QgisModelBaker/blob/master/.qgis-plugin-ci